src/Adapter/Tax/CommandHandler/ToggleTaxStatusHandler.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShop\PrestaShop\Adapter\Tax\CommandHandler;
+
+use PrestaShop\PrestaShop\Core\Domain\Tax\Command\ToggleTaxStatusCommand;
+use PrestaShop\PrestaShop\Core\Domain\Tax\CommandHandler\ToggleTaxStatusHandlerInterface;
+use PrestaShop\PrestaShop\Core\Domain\Tax\Exception\CannotToggleTaxStatusException;
+use PrestaShop\PrestaShop\Core\Domain\Tax\Exception\TaxException;
+use Tax;
+
+/**
+ * Class ToggleTaxStatusHandler handles command which toggles Tax status using legacy object model
+ */
+final class ToggleTaxStatusHandler extends AbstractTaxHandler implements ToggleTaxStatusHandlerInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function handle(ToggleTaxStatusCommand $command)
+    {
+        $taxIdValue = $command->getTaxId()->getValue();
+        $this->assertTaxWasFound($command->getTaxId(), $tax = new Tax($taxIdValue));
+
+        try {
+            if (false === $tax->toggleStatus()) {
```

Comment:
can you configure status in command? like you did in `BulkToggle`? 

the idea is that command should produce same result no matter how many times is was dispatched.